hydra-tui/src/Hydra/TUI.hs

```diff
@@ -580,9 +601,12 @@ draw Client{sk} CardanoClient{networkId} s =
 
   drawHeadState = case s of
     Disconnected{} -> emptyWidget
-    Connected{headState} ->
+    Connected{headState, pending} ->
       vBox
-        [ padLeftRight 1 $ txt "Head status: " <+> withAttr infoA (txt $ Prelude.head (words $ show headState))
+        [ padLeftRight 1 $
+            txt "Head status: "
+              <+> withAttr infoA (txt $ Prelude.head (words $ show headState))
+              <+> if pending then txt " (Transition pending)" else txt ""
```

Comment:
suggestion: `Transition pending` -> `Transaction pending` ?

Reply:
I would like to keep the wording as states+transitions. Not to interfere with transactions (to be submitted with L2)

Reply:
Fine by me, wasn't sure if it was 100% intentional